src/dirwatcher/dirwatcher.js

```diff
@@ -0,0 +1,52 @@
+import fs from 'fs';
+import EventEmitter from 'events';
+class MyEmitter extends EventEmitter{};
```

Comment:
What for?

Reply:
I updated my code to change the implementation.